### DISCUSSION TITLE
Signed commits

### DISCUSSION THREAD

**Discussion by csanders-rga** [BeanieODM/beanie#443] [Feature Request]
Hello there @roman-right, what are your thoughts on requiring signed commits for Beanie?  Our downstream fork requires signed commits, so we're having to do workarounds to pull from upstream.

Thanks for your time! :-)

**Comment by roman-right** [comment]
Hi,
I will check if it can sign commits when it squashes it on PRs or if I need a GH action for this.

**Reply by csanders-rga** [reply]
Hello, just checking on an update to this?  Thank you! :-)

**Reply by roman-right** [reply]
Hey! Sorry for the delay. I sorted this out. Since `1.18.0b1` it should work. The next non-beta version with signed commits will be `1.18.0`

**Reply by csanders-rga** [reply]
Thank you very much!
